src/System.Device.Gpio/System/Device/Gpio/BoardIdentification .cs

```diff
@@ -0,0 +1,276 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace System.Device.Gpio
+{
+    public class BoardIdentification
+    {
+        /// <summary>
+        /// The Raspberry Pi model.
+        /// </summary>
+        public enum Model
+        {
+            /// <summary>
+            /// Unknown model.
+            /// </summary>
+            Unknown,
+
+            /// <summary>
+            /// Raspberry Model A.
+            /// </summary>
+            RaspberryPiA,
+
+            /// <summary>
+            /// Model A+.
+            /// </summary>
+            RaspberryPiAPlus,
+
+            /// <summary>
+            /// Model B rev1.
+            /// </summary>
+            RaspberryPiBRev1,
+
+            /// <summary>
+            /// Model B rev2.
+            /// </summary>
+            RaspberryPiBRev2,
+
+            /// <summary>
+            /// Model B+.
+            /// </summary>
+            RaspberryPiBPlus,
+
+            /// <summary>
+            /// Compute module.
+            /// </summary>
+            RaspberryPiComputeModule,
+
+            /// <summary>
+            /// Pi 2 Model B.
+            /// </summary>
+            RaspberryPiB2,
+
+            /// <summary>
+            /// Pi Zero.
+            /// </summary>
+            RaspberryPiZero,
+
+            /// <summary>
+            /// Pi Zero W.
+            /// </summary>
+            RaspberryPiZeroW,
+
+            /// <summary>
+            /// Pi 3 Model B.
+            /// </summary>
+            RaspberryPiB3,
+
+            /// <summary>
+            /// Pi 3 Model B+.
+            /// </summary>
+            RaspberryPiB3Plus,
+
+            /// <summary>
+            /// Compute module 3.
+            /// </summary>
+            RaspberryPiComputeModule3,
+
+            /// <summary>
+            /// Pi 4 all versions.
+            /// </summary>
+            RaspberryPi4,
+        }
+        #region Fields
+
+        private readonly Dictionary<string, string> _settings;
+
+        private BoardIdentification(Dictionary<string, string> settings)
+        {
+            _settings = settings;
+        }
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// Get board model from firmware revision
+        /// </summary>
+        /// <see cref="http://www.raspberrypi-spy.co.uk/2012/09/checking-your-raspberry-pi-board-version/"/> for information.
```

Comment:
Do not use "see cref" for web links. This is what's breaking the build. (Don't you get an error in VS as well?)

Reply:
No I did not get any error. 
Ok, where is good place to keep these kind of info/refs?

Reply:
Ah, I see. It only generates a warning - another item on my desk. 
Just replace the line with 
```
/// See http://www.raspberrypi-spy.co.uk/2012/09/checking-your-raspberry-pi-board-version/ for information.
```
A <see cref= /> tag requires a reference to a valid code item.